frontend/app/globals.css

```
@@ -48,7 +50,7 @@
 body {
   background: var(--bg-primary);
   color: var(--text-secondary);
-  font-family: 'Inter', system-ui, sans-serif;
+  font-family: "Inter", system-ui, sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)